src/components/TaskCard/TaskCard.tsx

```diff
@@ -35,7 +36,7 @@ function FiledRequestCard({ filedRequestCardData, onDelete }: FiledRequestCard)
             <Box sx={styles.card__body}>
                 <Box sx={{ width: '60%' }}>
                     <Box>{description}</Box>
```

Comment:
A box is a div in mui, you should be putting a `Typography` inside the box to wrap your text